nautobot/extras/models/models.py

```diff
@@ -681,3 +685,35 @@ def log(
             else:
                 log_level = logging.INFO
             logger.log(log_level, str(message))
+
+
+@extras_features("graphql")
```

Comment:
Just checking - how does it behave when you query saved GraphQL queries through GraphQL/GraphiQL? Does the `query` body get escaped in some fashion, or what?

Reply:
Its returned back in an unformatted string.

```
{
  "data": {
    "query": [
      {
        "name": "test_1",
        "query": "query ($device: [String!]) {\n  devices(name: $device) {\n    config_context\n    name\n    position\n    serial\n    primary_ip4 {\n      id\n      primary_ip4_for {\n        id\n        name\n      }\n    }\n    tenant {\n      name\n    }\n    tags {\n      name\n      slug\n    }\n    device_role {\n      name\n    }\n    platform {\n      name\n      slug\n      manufacturer {\n        name\n      }\n      napalm_driver\n    }\n    site {\n      name\n      slug\n      vlans {\n        id\n        name\n        vid\n      }\n      vlan_groups {\n        id\n      }\n    }\n    interfaces {\n      description\n      mac_address\n      enabled\n      name\n      ip_addresses {\n        address\n        tags {\n          id\n        }\n      }\n      connected_circuit_termination {\n        circuit {\n          cid\n          commit_rate\n          provider {\n            name\n          }\n        }\n      }\n      tagged_vlans {\n        id\n      }\n      untagged_vlan {\n        id\n      }\n      cable {\n        termination_a_type\n        status {\n          name\n        }\n        color\n      }\n      tagged_vlans {\n        site {\n          name\n        }\n        id\n      }\n      tags {\n        id\n      }\n    }\n  }\n}"
      }
    ]
  }
}
```